platform-hub-api/spec/models/audit_spec.rb

```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
+
+RSpec.describe Audit, type: :model do
```

Comment:
Ah yeah, I initially started with a model unit test, but moved to the integration test instead.